pkg/controller/azure/database/claim.go

```diff
@@ -105,22 +119,36 @@ type MySQLInstanceClaimController struct{}
 
 // SetupWithManager adds a controller that reconciles MySQLInstance instance claims.
 func (c *MySQLInstanceClaimController) SetupWithManager(mgr ctrl.Manager) error {
+	name := strings.ToLower(fmt.Sprintf("%s.%s.%s",
+		databasev1alpha1.MySQLInstanceKind,
+		v1alpha2.MysqlServerKind,
+		v1alpha2.Group))
+
 	r := resource.NewClaimReconciler(mgr,
 		resource.ClaimKind(databasev1alpha1.MySQLInstanceGroupVersionKind),
-		resource.ClassKinds{Portable: databasev1alpha1.MySQLInstanceGroupVersionKind, NonPortable: v1alpha2.SQLServerClassGroupVersionKind},
+		resource.ClassKinds{
+			Portable:    databasev1alpha1.MySQLInstanceGroupVersionKind,
```

Comment:
Should be `databasev1alpha1.MySQLInstanceClassGroupVersionKind`

Reply:
This one is a good example of proposal in [comment](https://github.com/crossplaneio/stack-gcp/pull/24#discussion_r324423610) due to the fact that the correct `Portable` class kind is used in the predicates below. Still would be possible to use wrong kind, but would at least reduce having to make sure it is right twice :)